src/server/scripts/Northrend/Naxxramas/boss_grobbulus.cpp

```diff
@@ -101,22 +123,65 @@ struct boss_grobbulus : public BossAI
     }
 };
 
-struct npc_grobbulus_poison_cloud : public ScriptedAI
+struct npc_grobbulus_poison_cloud : public NullCreatureAI
 {
-    npc_grobbulus_poison_cloud(Creature* creature) : ScriptedAI(creature)
+    npc_grobbulus_poison_cloud(Creature* creature) : NullCreatureAI(creature) { }
+
+    void JustAppeared() override
+    {
+        DoCastSelf(SPELL_POISON_CLOUD_PASSIVE);
+        DoCastSelf(SPELL_PACIFY_SELF);
+        me->DespawnOrUnsummon(61s);
+    }
+};
+
+struct npc_fallout_slime : public ScriptedAI
+{
+    npc_fallout_slime(Creature* creature) : ScriptedAI(creature)
     {
-        SetCombatMovement(false);
-        creature->SetReactState(REACT_PASSIVE);
+        Initialize();
     }
 
-    void IsSummonedBy(WorldObject* /*summoner*/) override
+    void Initialize()
     {
-        // no visual when casting in ctor or Reset()
-        DoCast(me, SPELL_POISON_CLOUD_PASSIVE, true);
-        DoCast(me, SPELL_PACIFY_SELF, true);
+        me->SetCorpseDelay(2, true);
+        me->SetReactState(REACT_PASSIVE);
+        DoCastSelf(SPELL_DISEASE_CLOUD);
+
+        _scheduler.Schedule(2s, [this](TaskContext /*task*/)
+        {
+            me->SetReactState(REACT_AGGRESSIVE);
+
+            if (Unit* summoner = ObjectAccessor::GetUnit(*me, _summonerGUID))
+                // maybe only if threat list was empty?
+                AddThreat(summoner, 100000.0f);
```

Comment:
I recommend using the "FixateTarget" method, it's really cool.

Reply:
They are not fixated, just added 100k threat in one sniff but didn't in other(no clue why)
In movies they sometimes may move to another target(no clue why)